packages/tasit-contracts/decentraland/scripts/install.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+PROJECT_DIR=$1;
+
+if [ -z "$PROJECT_DIR" ];
+then
+    echo "Error! Use: ./$0 <tasit-contract-dir>";
+    exit;
+fi
+
+DECENTRALAND_DIR="$PROJECT_DIR/decentraland";
+REPOS="mana land marketplace-contracts";
+
+for repo in $REPOS;
+do
+    REPO_DIR="$DECENTRALAND_DIR/$repo";
+
+    if [ ! -e "$REPO_DIR/package.json" ];
+    then
+        rm -rf $REPO_DIR;
+        git clone https://github.com/decentraland/$repo.git $REPO_DIR;
```

Comment:
This should perhaps change to cloning from a particular release tag or branch

Reply:
Do you think this should get merging? That way we would make sure we're testing against the code versions that are deployed as opposed to the bleeding edge of their default GitHub branch.

I'm inclined to say it shouldn't gate merging, but we'll likely want to change this in a follow-on PR.

Reply:
Yes, you are right. Assuming that the last tag has the last version of deployed contract we should clone from there.
And tracking project/contracts to keep synchronized (refs https://github.com/tasitlabs/TasitSDK/issues/9 and https://github.com/tasitlabs/TasitSDK/issues/135)

Reply:
Okay. We can address this in those issues.